cis/plugins/validation/mozilliansorg_publisher_plugin.py

```diff
@@ -5,6 +5,10 @@
 utils.StructuredLogger(name=__name__, level=logging.INFO)
 logger = logging.getLogger(__name__)
 
+# Allows user creation by this publisher.
+CAN_CREATE_USER = True
+ENFORCE_ATTR_WHITELIST = False
```

Comment:
add XXX here too saying we need to turn this back on later (in fact mainly here)